### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by davidrijsman**
Documentation update maximum wait constraints and sequence generator example.

### CODE REVIEW

File: nextroute/model_constraint_maximum_wait_stop.go [hunk 1]
```diff
@@ -3,7 +3,9 @@ package nextroute
 import "github.com/nextmv-io/sdk/connect"
 
 // MaximumWaitStopConstraint is a constraint that limits the time a vehicle can
-// wait between two stops.
+// wait between two stops. Wait is defined as the time between starting a stop
+// and arriving at the location of the stop, [SolutionStop.StartValue()] -
```
Comment (larsbeck) [inline nextroute/model_constraint_maximum_wait_stop.go]: Should this be the other way around? I arrive first and then I start. This is just wording, but maybe that is a bit clearer.

File: nextroute/model_constraint_maximum_wait_stop.go [hunk 2]
```diff
@@ -12,9 +14,11 @@ type MaximumWaitStopConstraint interface {
 	Maximum() StopDurationExpression
 }
 
-// NewMaximumWaitStopConstraint returns a new MaximumWaitStopConstraint. The maximum
-// wait constraint limits the time a vehicle can wait between two stops. Wait
-// time is defined as the time a vehicle is neither moving nor serving a stop.
+// NewMaximumWaitStopConstraint returns a new MaximumWaitStopConstraint. The
+// maximum wait constraint limits the time a vehicle can wait between two stops.
+// Wait is defined as the time between starting a stop and arriving at the
+// location of the stop, [SolutionStop.StartValue()] -
```
Comment (larsbeck) [inline nextroute/model_constraint_maximum_wait_stop.go]: Same here

File: nextroute/model_constraint_maximum_wait_vehicle.go [hunk 3]
```diff
@@ -3,7 +3,9 @@ package nextroute
 import "github.com/nextmv-io/sdk/connect"
 
 // MaximumWaitVehicleConstraint is a constraint that limits the accumulated time
-// a vehicle can wait at stops on its route.
+// a vehicle can wait at stops on its route. Wait is defined as the
+// time between starting a stop and arriving at the location of the stop,
```
Comment (larsbeck) [inline nextroute/model_constraint_maximum_wait_vehicle.go]: Same here

File: nextroute/model_constraint_maximum_wait_vehicle.go [hunk 4]
```diff
@@ -14,8 +16,9 @@ type MaximumWaitVehicleConstraint interface {
 
 // NewMaximumWaitVehicleConstraint returns a new MaximumWaitVehicleConstraint.
 // The maximum wait constraint limits the accumulated time a vehicle can wait at
-// stops on its route. Wait time is defined as the time a vehicle is neither
-// moving nor serving a stop.
+// stops on its route. Wait time is defined as the time between starting a stop
+// and arriving at the location of the stop, [SolutionStop.StartValue()] -
```
Comment (larsbeck) [inline nextroute/model_constraint_maximum_wait_vehicle.go]: Same here

File: nextroute/model_constraint_maximum_wait_stop.go (hunk 1 above)
Comment (davidrijsman) [reply nextroute/model_constraint_maximum_wait_stop.go]: If you start at the time you arrive there is no waiting, starting later than arriving creates waiting time (at the gate for example).

File: nextroute/model_constraint_maximum_wait_stop.go (hunk 2 above)
Comment (davidrijsman) [reply nextroute/model_constraint_maximum_wait_stop.go]: If you start at the time you arrive there is no waiting, starting later than arriving creates waiting time (at the gate for example).

**Review by larsbeck — APPROVED** [review]

**Pull request merged**